README.md

```diff
@@ -53,7 +53,9 @@ document.addEventListener('readystatechange', e => {
 
 ---
 
+
 ## Built-in wallets, or passthrough wallets from your dApp
+**If your dApp already have a `<WalletProvider />`, it is a requirement to use `passthroughWallet` to ensure proper Wallet Standard support.**
```

Comment:
If your dApp already have a `<WalletProvider />`, it is a requirement to use `passthroughWallet` for the terminal to behave properly with your app.

We know it is bugged in other cases and UX is bad if someone happens to connect in the terminal.